data/supporters.json

```diff
@@ -1647,6 +1647,44 @@
           "email": "j.a.todorova@gmail.com"
         }
       ]
+    },
+    {
+      "name": "FOSS Backstage",
+      "city": "Berlin",
+      "country": "Germany",
+      "link": "https://foss-backstage.de",
+      "twitter": "fossbckstg",
+      "contacts": [
+        {
+          "name": "Ekaterina Klykova",
+          "email": "ekl@newthinking.de",
+          "phone": "+49 3092105978"
+        },
+         {
+          "name": "Paul Berschick",
+          "email": "pbe@newthinking.de",
+          "phone": "+49 3092105977"
+        },
```

Comment:
Only add a comma, when another opening `{` is following. The last item in JSON must not have a comma (in JavaScript this would be allowed though).